core/scripts/cli/cli_entry.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# This script should be installed with the UMA project and symlinked to the "uma" file in the npm binary directory.
```

Comment:
So according to the `bin` description in this [NPM guide](https://docs.npmjs.com/files/package.json), this `.sh` script will get symlinked automatically by npm. This is what you intend, correct?